applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_interface_coulomb_with_tension_cut_off.cpp

```diff
@@ -354,65 +354,56 @@ KRATOS_TEST_CASE_IN_SUITE(InterfaceCoulombWithTensionCutOff_Check, KratosGeoMech
     // Act & Assert
     KRATOS_EXPECT_EXCEPTION_IS_THROWN(
         [[maybe_unused]] const auto unused = law.Check(properties, element_geometry, process_info),
-        "Error: GEO_COHESION is not defined for property 3")
+        "GEO_COHESION does not exist in the property with Id 3.")
     properties.SetValue(GEO_COHESION, -1.0);
     KRATOS_EXPECT_EXCEPTION_IS_THROWN(
-        [[maybe_unused]] const auto unused = law.Check(properties, element_geometry, process_info),
-        "Error: value of GEO_COHESION for property 3 is out of range: -1 is not in [0.0, ->")
+        [[maybe_unused]] const auto unused = law.Check(properties, element_geometry, process_info), "GEO_COHESION in the property with Id 3 has an invalid value: -1 is out of the range [0; -].")
```

Comment:
My suggestion in the checkproperties class should change this (and probably other places) to 
```suggestion
        [[maybe_unused]] const auto unused = law.Check(properties, element_geometry, process_info), "GEO_COHESION in the property with Id 3 has an invalid value: -1 is out of the range [0; -).")
```